controller/v1/users.py

```
@@ -0,0 +1,169 @@
+    else:
+        # 신규 회원인 경우 Register Token 반환
+        return KakaoLoginResponse(
+            is_new_user=True,
+            register_token=result["register_token"]
+        )
+
+@router.post("/auth/kakao/register", response_model=Token)
+async def kakao_register(
+    response: Response,
+    data: KakaoRegisterRequest,
+    db: AsyncSession = Depends(get_db)
```

Ruff (0.15.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)